### ISSUE TITLE
v8 - When clicking preview, document gets published

### ISSUE THREAD

**Issue by tristanjthompson** [umbraco/Umbraco-CMS#3881]
### Bug summary
Found in v8
When editing a document, if you click "Preview", the document gets published before the preview window launches.  It should only be saved.

### Steps to reproduce
Steps to recreate:
* In the content tree, open a document to begin editing.
* Change a value in one of the fields
* Click the "Preview" button at the bottom-right - this should only save the content before previewing.
  * This will open the preview page in a new tab
* Switch back to the original tab and refresh the page
* Click on the "Info" app at the top-right
* *Bug demonstration* - You will see that the last action says "Publish - Content saved and Published" - it should say "Save - Content Saved"
* Other steps showing the issue:
  * Switch back to the preview pane and end the preview - the new content is still visible
  * Open an incognito window and load up the page you were previewing - you will see the new content
  * Reset IIS & delete ~/App_Data/NuCache.Content.db, then open the page you were previewing - you will see the new content.

### Expected result
If you click "Preview", it should save the content and then open preview mode.  You should see the _old_ content once you exit preview mode or open the page in a new incognito window.

### Actual result
When you click "Preview", it does a "Save & Publish" before opening the preview window.

**Comment by karltynan** [comment]
Agreed, I have seen this happening on the latest version of `temp8` (14/12/2018).

**Comment by zpqrtbnk** [comment]
reproduced

**Comment by kjac** [comment]
PR in #3891

**Comment by zpqrtbnk** [comment]
AWESOME - reviewing asap

**Comment by kjac** [comment]
Whoo! 🎉
